src/Microsoft.TestPlatform.Extensions.HtmlLogger/HtmlLogger.cs

```diff
@@ -153,12 +153,27 @@ public void TestMessageHandler(object sender, TestRunMessageEventArgs e)
             switch (e.Level)
             {
                 case TestMessageLevel.Informational:
+                    if(TestRunDetails.RunLevelMessageInformational == null)
+                    {
```

Comment:
Tests for this change ??

Reply:
tests were all ready added .. i thing those are sufficient.